packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/index.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { Story, Meta } from '@storybook/html';
+import storybookEmail, { StorybookEmailArgs } from '../../storybook-email';
+// const configURL = require('../../../../../config').get('contentServer.url')
+
+export default {
+  title: 'Emails/cadReminder',
+} as Meta;
+
+const Template: Story<StorybookEmailArgs> = (args) => storybookEmail(args);
+
+const commonPropsWithOverrides = (
```

Comment:
Added these overriding props but I'm not sure how much we can benefit from this and storybook/controls in general as we might not be able to have a separate control for the objects as we have for other argtypes. Linking [this PR](https://github.com/storybookjs/storybook/pull/12685) for context.

Reply:
Huh, I didn't dive much into Les' comment / how those auto generated controls work in Storybook 6.0+ when I suggested this function. I haven't played much with Storybook controls, but they do seem pretty neat. If it's not too much reworking, maybe we should move forward with that approach. 🙂